### DISCUSSION TITLE
Issue with Article Schema Markup

### DISCUSSION THREAD

**Discussion by Hariko50** [GetPublii/Publii#1701] [Help]
I'm experiencing a problem with the article schema in Publii. The Google Rich Results Test shows that the datePublished and dateModified fields are missing the time zone attribute. I've updated the template files to include the correct date format, including the time zone, but the issue remains unresolved. Has anyone else faced this problem, or does anyone know how to ensure that the date is correctly formatted for article schema in Publii? I use Simple Theme v. 3.0.

**Comment by bjazmoore** [comment]
I had never heard of Google Rich Results Test before.  I checked out one of my play / demo sites.  I have two posts in this site and several pages.  The page the flagged the error you are describing is my Homepage titled "Vacation in Space".  See attached:

![1](https://github.com/user-attachments/assets/063bf4ec-43b3-4aca-8c1c-3f8d1e78fb79)

**Comment by dziudek** [comment]
@Hariko50 @bjazmoore - as you can see it is not an error but a suggestion :) Anyway we will add improved output in v.0.46.2 for it

**Reply by bjazmoore** [reply]
> @Hariko50 @bjazmoore - as you can see it is not an error but a suggestion :) Anyway we will add improved output in v.0.46.2 for it

Thanks
